src/features/demo-mode/DemoLoginHint.tsx

```diff
@@ -12,7 +12,7 @@ export const DemoLoginHint = () => {
   const mockedUsername = 'admin';
   const mockedPassword = 'admin';
 
-  if (!process.env.NEXT_PUBLIC_IS_DEMO) return null;
+  if (process.env.NEXT_PUBLIC_IS_DEMO !== 'true') return null;
```

Comment:
I think it's a hotfix and we can do better (maybe by transforming it to a real boolean or at least doing a getter)

Reply:
strings are ok for env var to be honest, looks good to me like that